enh(discovery/service): add custom parameters with example (#616)
centreon/centreon-documentation: en/monitoring/discovery/administration.md

parameters:

```yaml
- id: service_discovery_poller_1
  timespec: "15 9 * * *"
  action: LAUNCHSERVICEDISCOVERY
  parameters:
    filter_pollers:
      - Poller-1
- id: service_discovery_poller_2_linux
  timespec: "30 9 * * *"
  action: LAUNCHSERVICEDISCOVERY
  parameters:
    filter_pollers:
      - Poller-2
    filter_rules:
      - OS-Linux-SNMP-Disk-Name
      - OS-Linux-SNMP-Traffic-Name
- id: service_discovery_poller_2_windows
  timespec: "45 9 * * *"
  action: LAUNCHSERVICEDISCOVERY
  parameters:
    filter_pollers:
      - Poller-2
    filter_rules:
      - OS-Windows-SNMP-Disk-Name
      - OS-Windows-SNMP-Traffic-Name
```

Here is the list of all available parameters:

| Key                  | Value                                                                                             |
|----------------------|---------------------------------------------------------------------------------------------------|
| filter\_rules        | Array of rules to use for discovery (empty means all)                                             |
| force\_rule          | Run disabled rules ('0': not forced, '1': forced)                                                 |

> | filter\_hosts        | Array of hosts against which run the discovery (empty means all)                                  |

Reviewer: ```suggestion
| filter\_hosts        | Array of hosts which will run the discovery (empty means all)                                  |
```

> | filter\_pollers      | Array of pollers for which linked hosts will be discovered against (empty means all)              |

Reviewer: ```suggestion
| filter\_pollers      | Array of pollers for which linked hosts will undergo discovery (empty means all)              |
```

| dry\_run             | Run discovery without configuration changes ('0': changes, '1': dry run)                          |

> | no\_generate\_config | No configuration generation (even if there is some changes) ('0': generation, '1': no generation) |

Reviewer: ```suggestion
| no\_generate\_config | No configuration generation (even if there are some changes) ('0': generation, '1': no generation) |
```
